src/test/java/com/salesforce/dropin/test/TestShadowPathHelper.java

```diff
@@ -0,0 +1,32 @@
+package com.salesforce.dropin.test;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import com.salesforce.selenium.support.findby.ShadowPathHelper;
+
+/**
+ * Tests for Shadow Path Helper functions
+ * 
+ * @author ytao
+ *
+ */
+public class TestShadowPathHelper {
+	
+	@Test
+	public void testShadowPath() {
```

Comment:
Great idea to have unit tests. In the helper class I suggested to introduce a conversion method for the revers plus a method renaming. Please update the test class accordingly.